legend-engine-shared-core/pom.xml

```diff
@@ -79,6 +79,13 @@
         </dependency>
         <!-- For Rest Result -->
 
+        <!-- For Request Context -->
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+        </dependency>
```

Comment:
We should not expose servlet related classes to plan execution to keep the execution agnostic of who triggered it.